packages/next-ui/TextInputNumber/TextInputNumber.tsx

```diff
@@ -163,7 +163,7 @@ export function TextInputNumber(props: TextInputNumberProps) {
       }}
       inputProps={{
         ...inputProps,
-        'aria-label': i18n._(/* i18n */ 'Number'),
+        'aria-label': i18n._(/* i18n */ 'Add to cart quantity'),
```

Comment:
Please change on the usage side, since this field can be used for more fields

Reply:
Done!